source/manual/zendesk.html.md

```diff
@@ -0,0 +1,88 @@
+line is required, assign them on the ticket and move on to the next ticket (keeping the ticket in the
+2nd line queue).
+
+Ignore 'Low' priority tickets - these have been triaged by 2nd line for GOV.UK product managers to review.
+
+New tickets are 'Normal' priority by default and require triaging (i.e. a developer reading and categorising
+it).
+
+## Triaging a ticket
+
+If you have any doubts as to the legitimacy or urgency of a ticket, it should be deferred to the delivery manager.
+
+Once you've established that a ticket is legitimate, you should follow [the process in this diagram](zendesk-triage-diagram).
+There is a printout on the 2nd line desk. In general:
+
+- Questions from users of data.gov.uk can be answered straight away (we're their first line of support)
+- General queries are usually handled by 1st line and content support
+- Product requests should be changed to 'Low' priority and assigned to the product owner of the most relevant
+  product (you can find ownership information under [Applications on GOV.UK](https://docs.publishing.service.gov.uk/apps.html))
+- Technical problems to investigate should be reprioritised as "High" and dealt with by 2nd line
+
+In all cases, 2nd line will respond to the ticket, even if it is simply a note to the user saying that
+you've passed their request to the relevent team.
+
+### Service Level Agreements
+
+GOV.UK have committed to a minimum service level agreement for 2nd line support.
+
+* 80% of tickets get a first reply to the ticket within 2 working days
+* 80% of general enquiries (public tickets) resolved within 5 working days
+* 70% of department tickets are closed within 5 working days
+
+If the volume of Zendesk tickets is overwhelming, talk to the delivery manager or a senior tech member
+for assistance.
+
+## Picking up a ticket
+
+Click the "take it" link under "Assignee", followed by "Submit as Open", to formally undertake a ticket.
+If you know you cannot solve a ticket immediately, fill in a "Public reply" to the requester to let them
+know you’re looking into it.
+
+Before attempting to solve the problem yourself, read [Common 2nd line support requests for data.gov.uk](/manual/data-gov-uk-2nd-line.html),
+which outlines some of the common tickets that come in and how to fix them.
+
+You can use Internal Notes to keep a log of actions you've taken so far: this can make it easier for other
+staff to pick up where you left off if you're unable to solve the ticket yourself. Please note that departments
+can see internal notes, so don’t write anything you wouldn’t say to someone publicly.
+
+If you need more information from the user, or confirmation that everything is now working as expected, fill
+in the "Public reply" as appropriate and click "Submit as Pending".
+
+## Closing a ticket
+
+Once you've resolved a ticket, click "Submit as Solved".
+
+Note that if a ticket has been resolved and pending a response for 5 or more days with no response from the
```

Comment:
This feels like it should be under it's own heading, e.g. `Tickets left pending without a response` (or something like that)